test/benchmark/bench_linux_and_aster.sh

```diff
@@ -57,7 +59,11 @@ run_benchmark() {
     echo "Preparing libraries..."
     prepare_libs
 
-    local asterinas_cmd="make run BENCHMARK=${benchmark} ENABLE_KVM=1 RELEASE_LTO=1 2>&1"
+    local aster_scheme_cmd=""
+    if [ -n "$aster_scheme" ]; then
+        aster_scheme_cmd="SCHEME=${aster_scheme}"
+    fi
+    local asterinas_cmd="make run BENCHMARK=${benchmark} ${aster_scheme_cmd} ENABLE_KVM=1 RELEASE_LTO=1 2>&1"
     local linux_cmd="/usr/local/qemu/bin/qemu-system-x86_64 \
```

Comment:
Should Linux takes iommu into QEMU arguements?

Reply:
I prefer not to do this for now, as enabling IOMMU for Asterinas is primarily for security reasons, and we don't currently require Linux to do the same. Additionally, this would complicate the boot parameters for Linux.